zio-sqs/src/test/scala/zio/sqs/ZioSqsMockServer.scala

```diff
@@ -31,3 +32,45 @@ object ZioSqsMockServer extends TheSQSRestServerBuilder(None, None, "", 9324, No
         .endpointOverride(uri)
     )
 }
+
+object MockSqsServerAndClient {
+
+  lazy val layer: ZLayer[Any, Throwable, Sqs] =
```

Comment:
I wanted to suggest this approach, where a new mock server is created, with a random port to allow for parallel testing.
The was config and sqs client are made all together, so that tests just need this single layer and no setup within the tests themselves.